x-pack/plugins/index_management/__jest__/client_integration/home/home.test.ts

```diff
@@ -84,5 +87,30 @@ describe('<IndexManagementHome />', () => {
         expect(exists('templateList')).toBe(true);
       });
     });
+    describe('toggles', () => {
+      it('Should show no indices to show when there are no indices ', () => {
+        const { find } = testBed;
+
+        const indicesListTable = find('indicesList');
+        const noIndicesMessageText = indicesListTable
+          .find('[data-test-subj="noIndicesMessage"]')
+          .text();
+        expect(noIndicesMessageText).toEqual('No indices to show');
+      });
+
+      it('Should show system indices if hidden indices is toggled. ', async () => {
```

Comment:
I think the `indices_tab.test.tsx` file is a better location for any tests specific to the indices tab. There's even a test there that is related to this toggle: https://github.com/elastic/kibana/blob/master/x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts#L47

I think this file is mostly for tests for everything outside of that tab content area.

Reply:
Moved and updated. We were on the same page as I moved it and then came across this message. :D